### ISSUE TITLE
rainbow fails to build on 10.10.3

### ISSUE THREAD

**Issue by bfontaine** [Homebrew/homebrew-science#2550]
```
==> Installing rainbow from homebrew/homebrew-science
==> Downloading https://downloads.sourceforge.net/project/bio-rainbow/rainbow_2.0.2.tar.gz
==> Downloading from http://heanet.dl.sourceforge.net/project/bio-rainbow/rainbow_2.0.2.tar.gz
######################################################################## 100,0%
==> make
clang: error: cannot specify -o when generating multiple output files
make: *** [rbasm] Error 1
make: *** Waiting for unfinished jobs....
make: *** [rbmergetag] Error 1
make: *** [rainbow] Error 1
```

Logs: https://gist.github.com/4fe227b01a15e7ca45ca

**Comment by dunn** [comment]
I ran into this when trying to update to 2.0.3, this only happens with Clang.

**Comment by dpo** [comment]
That seems bogus. It should probably be reported upstream.

**Comment by dunn** [comment]
https://sourceforge.net/p/bio-rainbow/discussion/general/thread/f29dda20/

**Comment by dpo** [comment]
Thanks!

**Comment by iMichka** [comment]
Fixed with rainbow 2.0.4 (#5838)
